apps/dashboard/app/controllers/application_controller.rb

```diff
@@ -57,7 +58,7 @@ def sys_app_groups
   end
 
   def pinned_app_group
-    OodAppGroup.groups_for(apps: @pinned_apps, nav_limit: @user_configuration.pinned_apps_menu_length)
+    OodAppGroup.groups_for(apps: @pinned_apps, group_by: :feature, nav_limit: @user_configuration.pinned_apps_menu_length)
   end
```

Comment:
This is ^^the idea to solve the `categories` / `original_categories` issue fro `FeatureApp`